### ISSUE TITLE
[Suggestion] "Pretty" display for food item saturation

### ISSUE THREAD

**Issue by MarioSMB** [squeek502/AppleSkin#259]
As a player I find it far less important to know the exact amount of saturation a food item gives, it would be nice to have a rough indicator of how much saturation an item will provide, ala Quark's "Light Meal"/"Fulfilling Meal".

**Comment by LoganTheAroAce** [comment]
you can disable the hunger and saturation display with appleskin and install either quark or a mod that adds just the "Light Meal"/"Fulfilling Meal". Still, This is a good suggestion, and they should add this in the future

**Comment by MarioSMB** [comment]
Indeed, besides Quark ~~(which has been abandoned as of writing)~~ there does not seem to be another mod that does this, which is a shame as it is a very convenient way of displaying the average gains from food items without being too verbose.
